config/config.yaml

```diff
@@ -127,6 +127,10 @@ placement_policy:
   # Region to placement policy mapping json file.
   # Path to container policy mapping. The same as '--container-policy' flag for authmate
   region_mapping: /path/to/container/policy.json
+  regions:
```

Comment:
Maybe call it `locations`, a bit more neutral? It's `LocationConstraint anyway`.

Reply:
Renamed